src/engraving/libmscore/keysig.cpp

```diff
@@ -313,8 +313,22 @@ void KeySig::draw(mu::draw::Painter* painter) const
 {
     TRACE_OBJ_DRAW;
     painter->setPen(curColor());
+    qreal _spatium = spatium();
+    int lines = staff() ? staff()->staffTypeForElement(this)->lines() : 5;
+    qreal lineDist = staff() ? staff()->staffTypeForElement(this)->lineDistance().val() : 1;
     for (const KeySym& ks: _sig.keySymbols()) {
         drawSymbol(ks.sym, painter, PointF(ks.pos.x(), ks.pos.y()));
+        // draw ledgerLines
+        qreal x = ks.pos.x() - ((ks.sym == SymId::accidentalSharp) ? (_spatium * .15) : (_spatium * .25));
+        int i = int(ks.spos.y() / lineDist);
```

Comment:
This kind of casts are frowned upon, in master at least, better use the more verbose `static_cast<int>(...)`